packaging/debian/rules

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/make -f
 
+export PREFIX=/usr
```

Comment:
This is weird.  It should be passed as an arg to `make`.  Is that why you used `?=` ?

Reply:
I used `?=` as that's coming from starter-kit, and it intuitively feels more correct. But keeping this local to the install rule is nicer indeed, I changed it.